package/MDAnalysis/lib/formats/include/readdcd.h

```diff
@@ -136,7 +136,7 @@ static int write_dcdheader(fio_fd fd, const char *remarks, int natoms,
  * Output: 0 on success, negative error code on failure.
  * Side effects: coordinates are written to the dcd file.
  */
-static int write_dcdstep(fio_fd fd, int curstep, int curframe, 
```

Comment:
The first version of the code came from catdcd , later versions (?) from the VMD molfile plugin code, which is published under the UIUC  open source licence (see our file LICENSES), which is GPL compatible. 

# VMD sources
[Getting the full source tree from the UIUC CVS](http://www.ks.uiuc.edu/Research/vmd/doxygen/cvsget.html) (!) is cumbersome and you need to [request](http://www.ks.uiuc.edu/Research/vmd/cvsrequest.html) access (I don't think they would like you to make a git mirror, and the [current VMD license] (http://www.ks.uiuc.edu/Research/vmd/current/LICENSE.html)forbids it). Furthermore, now looking at the [current VMD license](http://www.ks.uiuc.edu/Research/vmd/current/LICENSE.html) it does not qualify as open. 

# Plugins
However, the *plugins* have a different license: [UIUC Open Source License](http://www.ks.uiuc.edu/Research/vmd/plugins/pluginlicense.html): that one is open.

Although hidden in CVS, the [plugin files](http://www.ks.uiuc.edu/Research/vmd/plugins/doxygen/filestruct.html) are visible online, e.g., the source pdcdplugin.c](http://www.ks.uiuc.edu/Research/vmd/plugins/doxygen/dcdplugin_8c-source.html).